### PR TITLE
requirements: forcing dojson 0.4.0

### PR DESCRIPTION
**Pull request by kaplun**
Signed-off-by: Samuele Kaplun <samuele.kaplun@cern.ch>

### CODE REVIEW

File: base.requirements.txt [hunk 1]
```diff
@@ -1,4 +1,5 @@
 ## Base requirements
+dojson==0.4.0
```
Comment (jmartinm) [inline base.requirements.txt]: shall we add here https://github.com/inspirehep/inspire-next/blob/master/setup.py#L32 instead?

Comment (kaplun) [inline base.requirements.txt]: Is that line ever run? We don't do `pip install .` or python setup.py build`, at least on Travis

Comment (jmartinm) [inline base.requirements.txt]: Good question. @jalavik ?

Comment (jalavik) [inline base.requirements.txt]: It is picked up when `-e .` is run as part of `requirements.txt`, so I agree with @jmartinm  that it should be pinned in setup.py.

Comment (jalavik) [inline base.requirements.txt]: requirements file entries should only be for git repo requirements.

**Pull request merged**